packages/core/core/src/ParcelConfig.js

```diff
@@ -14,44 +13,47 @@ import type {
   Optimizer,
   Reporter,
   Validator,
+  PreProcessedParcelConfig,
+  ParcelPluginNode,
+  PureParcelConfigPipeline,
+  ExtendableParcelConfigPipeline,
```

Comment:
Should these types be in the internal core types rather than the public types?

Reply:
I'lll move them